core/src/main/java/org/elasticsearch/search/suggest/completion/context/ContextMappings.java

```diff
@@ -81,7 +81,7 @@ public int size() {
     public ContextMapping get(String name) {
         ContextMapping contextMapping = contextNameMap.get(name);
         if (contextMapping == null) {
-            throw new IllegalArgumentException("Unknown context name[" + name + "], must be one of " + contextNameMap.size());
+            throw new IllegalArgumentException("Unknown context name[" + name + "], must be one of " + contextNameMap.keySet().toString());
```

Comment:
I think we'd better sort the list of keys before adding it to the string so we get consistent error messages. This'll make testing much easier and be marginally easier on the eyes.